docs/service-catalog/docs/014-details-etcd-database.md

```diff
@@ -95,14 +102,14 @@ kubectl delete -f assets/etcd-restore/operator-deploy.yaml
 [etcd-backups]:https://github.com/coreos/etcd-operator/blob/master/doc/user/walkthrough/backup-operator.md
 [etcd-restores]:https://github.com/coreos/etcd-operator/blob/master/doc/user/walkthrough/restore-operator.md
 
-[sc-etcd-sub-chart]:../../../resources/core/charts/service-catalog/charts/etcd/templates/etcd-cluster-cr.yaml
-[sc-etcd-cluster-no]:../../../resources/core/charts/service-catalog/charts/etcd/templates/etcd-cluster-cr.yaml#L6
-[sc-backup-sub-chart]:../../../resources/core/charts/service-catalog/charts/etcd/templates/backup-job.yaml
-[etcd-operator-chart]:../../../resources/core/charts/service-catalog/charts/etcd
-[etcd-backup-operator-chart]:../../../resources/core/charts/etcd-operator/templates/backup-deployment.yaml
-[core-chart-values]:../../../resources/core/values.yaml
+<!-- This absolute paths should be repalced with then relative links after adding this functionality to kyma -->
```

Comment:
These absolute paths should be replaced with the relative links after adding this functionality to Kyma